src/_shared/components/HandleApiResponse/HandleApiResponse.tsx

```diff
@@ -1,29 +1,14 @@
 import React, { useState } from 'react';
 import { ApolloError } from '@apollo/client';
-import { makeStyles } from '@material-ui/core/styles';
 import {
+  Alert,
   Box,
   CircularProgress,
   Collapse,
   Grow,
   IconButton,
-} from '@material-ui/core';
-import { Alert } from '@material-ui/lab';
-import CloseIcon from '@material-ui/icons/Close';
-
-/**
- * Styles for the error message alert.
- */
-const useStyles = makeStyles(() => ({
-  root: {
-    fontWeight: 400,
-    fontSize: '0.875rem',
-  },
-  title: {
-    fontWeight: 400,
-    fontSize: '1.125rem',
-  },
```

Comment:
This bit here wasn't even used! It is satisfying to know that the upgrade to MUI 5 can have PRs with net code lines deleted 🔥